### PR TITLE
Clarify a couple of steps in external-oasis-repos

### PR DESCRIPTION
**Pull request by DrDaveD**
Based on feedback from someone who used the instructions.

### CODE REVIEW

File: docs/data/external-oasis-repos.md [hunk 1]
```diff
@@ -179,7 +179,9 @@ In order to host a repository on OASIS, perform the following steps:
     representative before continuing with the remaining instructions; for all other repositories (such as `*.egi.eu`),
     you are done.
 
-1. One you are told in the ticket to proceed to the next step, execute the following commands:
+1. When you are told in the ticket to proceed to the next step, make sure that
+    the repository is not in a transaction (use `cvmfs_server abort` if not
+    sure) and execute the following commands:
```
Comment (brianhlin) [inline docs/data/external-oasis-repos.md:184]: Could we separate the `cvmfs_server abort` bit into its own step? It seems logically separate enough. Also, is there a command that can be run to detect if the repo is in a transaction so they can run the `abort` based on that information?

Comment (DrDaveD) [reply docs/data/external-oasis-repos.md:184]: Unfortunately there's not a straightforward way to detect if a repo is in transaction.  The abort could be in a separate step, I guess I'll do that.

**Comment by DrDaveD** [comment]
@brianhlin I see inconsistency in yellow-highlighted lines.  Which lines are supposed to be highlighted?

**Comment by brianhlin** [comment]
Could you upload a screenshot?

**Comment by DrDaveD** [comment]
> Could you upload a screenshot?

I mean for example step 4 under [Hosting a repository](https://opensciencegrid.org/docs/data/external-oasis-repos/#hosting-a-repository-on-oasis) has lines 1, 2, and 4 highlighted but not line 3.  Why  is that, what are the rules?  It's apparently not all command lines.  In those cases those 3 have <> substitutions, but there are other such things on the page that do not have them such as the cron entry in step 5.

**Comment by brianhlin** [comment]
Yeah, the intention was to highlight lines where the user would need to substitute their own values so that cron line could use it. Feel free to submit a separate PR for that

**Pull request merged**